src/averageAges.js

```diff
@@ -1,64 +1,71 @@
 'use strict';
 
 /**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+
+function findMother(people, person) {
+  return people.find(p => p.name === person.mother);
+}
```

Comment:
Try not to shorten the names to unreadable ones and don't forget to use destructuring for objects.
```suggestion
function findMother(people, person) {
  return people.find(({ name }) => name === person.mother);
}
```

Reply:
Yeah, you can use destructuring, or just a better name for a variable as `p` isn't informative.